src/locales/en/en.help.json

```diff
@@ -44,6 +44,23 @@
         "synonyminput": "Synonym"
       }
     },
+    "panelResult": {
+      "example": "Here is an example",
+      "part1a": " shows all Symbols and Synonyms found and not found from ",
+      "part1b": "in the panel.",
+      "part2": "The user input (leftmost column) may match a gene symbol, fusion, intron, or a synonym in each panel (column)",
+      "part3": {
+        "exactMatch": "has the exact ",
+        "anyMatch": "has a",
```

Comment:
Does not work with Intron (has a Intron instead of an Intron)

Reply:
done